README.md

```diff
@@ -33,7 +34,8 @@ In order to create a trigger that reacts when messages are posted to an unauthen
 |---|---|---|
 |brokers|JSON Array of Strings|This parameter is an array of `<host>:<port>` strings which comprise the brokers in your Message Hub instance|
 |topic|String|The topic you would like the trigger to listen to|
-|isJSONData|Boolean (Optional - default=false)|When set to `true` this will cause the feed to attempt the message content as JSON before passing it along as the trigger payload.|
+|isJSONData|Boolean (Optional - default=false)|When set to `true` this will cause the provider to attempt to parse the message content as JSON before passing it along as the trigger payload.|
+|isBase64Data|Boolean (Optional - default=false)|When set to `true` this will cause the provider to attempt to decode the message content as Base64 before passing it along as the trigger payload.|
```

Comment:
As useful as this might be, it is the opposite of the issue I have been reporting in #125. The flag should cause the feed to *encode* incoming binary data as base64, so that it can be passed as part of a valid JSON to the action.